src/components/v2/V2Create/tabs/ReviewDeployTab/DeployProjectButton.tsx

```diff
@@ -32,7 +32,7 @@ const PROJECT_ID_TOPIC_IDX = 3
  */
 const getProjectIdFromReceipt = (txReceipt: TransactionReceipt): number => {
   const projectIdHex =
-    txReceipt.logs[CREATE_EVENT_IDX]?.topics?.[PROJECT_ID_TOPIC_IDX]
+    txReceipt?.logs[CREATE_EVENT_IDX]?.topics?.[PROJECT_ID_TOPIC_IDX]
```

Comment:
This is a safeguard against this error:

<img width="1049" alt="Screen Shot 2022-05-07 at 6 50 44 pm" src="https://user-images.githubusercontent.com/96150256/167296755-81477229-0826-4511-a0f6-e437c6499a98.png">